### ISSUE TITLE
Is there any way to remove columnFilterContextMenu

### ISSUE THREAD

**Issue by QiaoLi1996** [inovua/reactdatagrid#238]
My users feel that the cost of use columnFilterContextMenu is too high, so they do not want to use the integrated menu of filters. I want to use the defaultFilterValue configuration to meet their basic filters needs.
![image](https://user-images.githubusercontent.com/73815759/153365877-85375356-0e71-4d42-a3d1-31ef50466a3b.png)

So，is there any way to not display columnFilterContextMenu , it is best to keep the icon, click on the icon and there is no response. I checked the documentation and some properties can't do this
![image](https://user-images.githubusercontent.com/73815759/153366211-c5f476b4-9b59-4cd7-88ec-7fcdac3f87ee.png)

**Comment by QiaoLi1996** [comment]
Can anyone offer some help？

**Comment by MuhammadSawalhy** [comment]
I came here to create an issue for that, but suddenly an idea came to me...

```css
.my-custom-filter .InovuaReactDataGrid__column-header__filter-settings {
  display: none;
}
```

But you can try to find the correct selector for your case.

**Comment by QiaoLi1996** [comment]
@MuhammadSawalhy Thank you very much, it works for me.
````
const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      '& .inovua-react-toolkit-menu': {
        display: 'none'
      }
    },
    column: {
      '& ~ .InovuaReactDataGrid__column-header__filter-wrapper': {
        flexFlow: 'row-reverse!important',
        '&  .InovuaReactDataGrid__column-header__filter-settings': {
          marginLeft: '4',
          marginRight: '8'
        }
      }
    }
  }));
````
![image](https://user-images.githubusercontent.com/73815759/155470346-a2b80002-c7dc-4beb-a10c-ed06d55db2eb.png)

**Comment by inovua-admin** [comment]
We fixed this in `v4.14.0` by adding a prop, [enableColumnFilterContextMenu](https://reactdatagrid.io/docs/api-reference#props-enableColumnFilterContextMenu), which can be set at the grid level or at the column level, via [columns.enableColumnFilterContextMenu](https://reactdatagrid.io/docs/api-reference#props-columns-enableColumnFilterContextMenu) of the grid.
